src/oom/oomd-manager.c

```diff
@@ -799,10 +799,15 @@ int manager_get_dump_string(Manager *m, char **ret) {
         _cleanup_(memstream_done) MemStream ms = {};
         OomdCGroupContext *c;
         FILE *f;
+        int r;
 
         assert(m);
         assert(ret);
 
+        r = oomd_system_context_acquire("/proc/meminfo", &m->system_context);
```

Comment:
maybe add a comment here along the lines of `/* always reread memory/swap info here. If swap monitoring is off we'll otherwise no have know it yet, and let's make sure to always report up-to-date data */`

or something like that